docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -351,9 +351,7 @@ Further examples can be found in the [Gatsby Image docs](https://github.com/gats
 
 Gatsby v2 removed `postcss-cssnext` and `postcss-import` from the default postcss setup.
 
-Use [`onCreateWebpackConfig`](/docs/add-custom-webpack-config) to specify your postcss plugins.
-
-Note: there will be a `postcss` plugin that allows you to configure postcss from a standard postcss config file. [Follow this discussion on issue 3284](https://github.com/gatsbyjs/gatsby/issues/3284).
+Use [`gatsby-plugin-postcss`](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-postcss) to specify your particular PostCSS configuration.
```

Comment:
Could you add the postcss configuration to replicate how Gatsby v1 is setup? That'll make it really easy to migrate. So the instructions for the npm install + the postcss.config.js file to add. 

https://github.com/gatsbyjs/gatsby/blob/389c829efc6797bf708acee8a56c4930aeb63e70/packages/gatsby/src/utils/webpack.config.js#L454

Reply:
@KyleAMathews Yes, sure. Thanks for the suggestion.

Reply:
@KyleAMathews Done. Could you please to review it again?